### PR TITLE
Translate calendar management forms

### PR DESCRIPTION
**Pull request by iant**

### CODE REVIEW

File: app/views/calendars/_current_events.html.erb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+      <th><%= t('calendars.current_events.type') %></th>
+      <th><%= t('calendars.current_events.start_date') %></th>
+      <th><%= t('calendars.current_events.end_date') %></th>
+      <th><%= t('calendars.current_events.last_updated') %></th>
       <% if can? :manage, calendar %>
-        <th>Status</th>
+        <th><%= t('calendars.current_events.status') %></th>
       <% end %>
-      <th>Actions</th>
+      <th><%= t('calendars.current_events.actions') %></th>
```
Comment (urbanwide) [inline app/views/calendars/_current_events.html.erb:11]: Could use common label here?

File: app/views/calendars/calendar_events/index.html.erb [hunk 2]
```diff
@@ -19,14 +19,14 @@
...
+            <th><%= t('calendars.current_events.type') %></th>
+            <th><%= t('calendars.current_events.start_date') %></th>
+            <th><%= t('calendars.current_events.end_date') %></th>
             <th>Last Updated</th>
             <% if can? :manage, @calendar %>
-              <th>Status</th>
+              <th><%= t('calendars.current_events.status') %></th>
             <% end %>
-          <th>Actions</th>
+          <th><%= t('calendars.current_events.actions') %></th>
```
Comment (urbanwide) [inline app/views/calendars/calendar_events/index.html.erb:29]: Common label possibly?

**Review by urbanwide — COMMENTED** [review]
Looks good @iant - couple of tiny comments

File: app/views/calendars/_current_events.html.erb (hunk 1 above)
Comment (iant) [reply app/views/calendars/_current_events.html.erb]: :+1: have pushed an update to use common labels

File: app/views/calendars/calendar_events/index.html.erb (hunk 2 above)
Comment (iant) [reply app/views/calendars/calendar_events/index.html.erb]: :+1: have pushed an update to use common labels

**Review by urbanwide — APPROVED** [review]
Looks good @iant 👍

**Pull request merged**